docs/dashboard/access-roles.md

```diff
@@ -0,0 +1,53 @@
+<!--
+title: Serverless Dashboard - Access Roles
+menuText: Access Roles
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Access Roles
+
+You can use the Serverless Framework Dashboard to set up an AWS Access Role to help you secure your service deployments on AWS by enabling the Serverless Framework to issue temporary AWS Access Keys to deploy your services to AWS.
+
+If an Access Role is not configured, the Serverless Framework service will use AWS Access Keys stored in [environment variables](https://serverless.com/framework/docs/providers/aws/guide/credentials/) or [AWS Profiles](https://serverless.com/framework/docs/providers/aws/guide/credentials/) to deploy your service. 
+
+With AWS Access Roles the AWS Access Keys are generated by Serverless Framework on every command and the credentials expire after one hour.  The Serverless Framework leverages AWS Security Token Service and the AssumeRole API to automate creating and usage of temporary credentials, so your developers can stay productive and work securely without doing this manually.
+
+If you do not use the Serverless Framework Dashboard to set up an AWS Access Role, then you will need to configure your Serverless Framework open source CLI to use the AWS Access Keys stored in [environment variables](https://serverless.com/framework/docs/providers/aws/guide/credentials/) or [AWS Profiles](https://serverless.com/framework/docs/providers/aws/guide/credentials/). 
+
+## Link your AWS Account
+
+1. Open https://dashboard.serverless.com/
+2. Once logged in, click "**profiles**" near the top of the page.
+3. Navigate to the profile you would like to configure with the AWS Access Role.
+4. In the **AWS credential access role** tab, expand the "_how to add a role_".
+5. Follow the directions which will take you through creating an IAM Role for the Serverless Framework.
+6. Click "**save changes**" in the deployment profile to save the IAM Role ARN to the profile.
+
+## Test your configuration
+
+Before you run `serverless deploy` ensure that the Serverless Framework open-source CLI is resolving the new AWS Access Keys from the Serverless Framework Dashboard.
+
+```
+serverless info
```

Comment:
Will this also work if the user isn't in a service directory / there's no service deployed? Was this behavior added to the the enterprise plugin?

Also /ccing @dschep here.

Reply:
nope. `sls info` only works in the directory of a deployed service.

Reply:
In that case we might want to update this section here.

Is there any other command which doesn't require a deployment up front? `serverless print` comes to my mind, but I'm not sure if this requires a proper AWS setup.

@alexdebrie since you've implemented it, does it check for a valid AWS setup (since it might have to deal with `ssm` variables)?

Reply:
@pmuens Yep, you're right that `serverless print` would not require a previous deployment.

It would only check for a valid AWS setup if you used a variable provider that uses AWS (SSM / CF)

Reply:
@skierkowski any suggested changes based on the above?

Reply:
Thanks for chiming in and clarifying the implementation here @alexdebrie :+1: